sdk/lusid/utilities/api_configuration.py

```diff
@@ -1,10 +1,7 @@
-import re
-
-
 class ApiConfiguration:
 
     def __init__(self, token_url=None, api_url=None, username=None, password=None, client_id=None, client_secret=None,
-                 app_name=None, certificate_filename=None, proxy_config=None):
+                 app_name=None, certificate_filename=None, proxy_config=None,flag_to_use_decimal=None):
```

Comment:
The more I think about this, the less it makes sense to have this as part of the `ApiConfiguration` I think we should follow the pattern set by the `tcp_keep_alive` setting which is to just pass it in when you construct your ApiClientFactory. 

I don't think there is any value having this come from a secrets file or environment variable.